File: LeetCode/Easy/0598-range-addition-ii/0598-range-addition-ii.java
class Solution {
    public int maxCount(int m, int n, int[][] ops) {

        if(ops.length == 0) return m * n;

        int leftMin = m;
        int rightMin = n;
        
        for(int i = 0; i < ops.length; ++i) {
            leftMin = Math.min(leftMin, ops[i][0]);
            rightMin = Math.min(rightMin, ops[i][1]);
        }

        return leftMin * rightMin;
    }
}

File: LeetCode/Easy/0598-range-addition-ii/0598-range-addition-ii.kt
class Solution {
    fun maxCount(m: Int, n: Int, ops: Array<IntArray>): Int {
        
        if(ops.size == 0) return m * n
        
        var leftMin = m
        var rightMin = n
        
        for(i in 0 .. ops.size - 1) {
            leftMin = Math.min(leftMin, ops[i][0])
            rightMin = Math.min(rightMin, ops[i][1])
        }
        return leftMin * rightMin
    }
}